docs/proposals/scheduledjob.md

```diff
@@ -216,6 +208,144 @@ In the above example:
   July 7th, 2pm.  This value will be validated according to the same rules which
   apply to `.spec.schedule`.
 
+## Fields Added to Job Template
+
+When the controller creates a Job from the JobTemplateSpec in the ScheduledJob, it
+adds the following fields to the Job:
+
+- a name, based on the ScheduledJob's name, but with a suffix to distinguish
+  multiple executions, which may overlap.
+- annotation with the nominal start time of the job, so that UIs and CLIs can show the
+  nominal start times of Jobs in relation to their parent, and so that the
+  controller can distinguish whether an existing job is started as
+  the most recent or a prior execution time (which may not be clear in the case
+  of controller-manager failures).  The annotation key will be 
+  `scheduledjob.kubernetes.io/nominal-start-time`.  The annotation value will be the RFC_3339
+  time and date of the intended start. 
```

Comment:
CreationTimestamp won't be exactly the same as the nominal start time.  Nominal start time might be `2016-05-19T15:00-08:00`, whereas CreationTimestamp might be `2016-05-19T15:01-08:00` if there was a bit of a delay in the controller or the apiserver.  Rather than requiring clients to decide how to round CreationTimestamp, it seems better to provide the rounded value.